src/helm/benchmark/run_expander.py

```diff
@@ -271,7 +271,7 @@ def expand(self, run_spec: RunSpec) -> List[RunSpec]:
                 name=run_spec.name,
                 adapter_spec=replace(
                     run_spec.adapter_spec,
-                    global_prefix=self.prefix,
+                    input_prefix=self.prefix,
```

Comment:
I changed this to add "Human" to every in-context example, which seemed more natural.  (cc @farzaank @JosselinSomervilleRoberts )